networktest.go

```diff
@@ -984,3 +984,61 @@ func (n *networkHarness) SendCoins(ctx context.Context, amt btcutil.Amount,
 		}
 	}
 }
+
+// CopyLogs copy/dumps Alice and Bob lnd daemon logs with specified period
+// of time in temporary directory.
+// NOTE: Panics error logs will not be logged.
+func (n *networkHarness) CopyLogs(period time.Duration) (func(), <-chan error) {
```

Comment:
What's the purpose of this method? It doesn't look to be used anywhere with the PR currently.